libs/analytics/README.md

```diff
@@ -1,45 +1,99 @@
 # Analytics Actions Mapping
 
-## VIEW_ITEM_LIST
 
-- DaffCategoryLoadSuccess
-- DaffCategoryPageLoadSuccess
-- DaffProductPageLoadSuccess (upsell/related)
 
-## VIEW_ITEM
 
-- DaffProductPageLoadSuccess
+**Before going deep down first we need to understand this technical term so Let's begin with the word *Analytics Action Mapping* so you can easily understand rest of the things.**
```

Comment:
Avoid using the first person in documentation: https://engineering.usu.edu/students/ewc/writing-resources/technical-writing-standards#:~:text=Narration%20(Point%20of%20View)&text=Third%20person%20narration%20is%20most,(common%20in%20business%20correspondence).